### PR TITLE
Feat/CU-2y2ptjv

### PR DESCRIPTION
**Pull request by danielanthonyl**
#### Todo

- [ ] unit tests(xx%)
- [ ] styles (xx%)
- [ ] codacy report (xx%)
- [ ] typescript (xx%)
- [ ] logic (xx%)

### What are the plans if anything goes wrong?

- roll off the last merge

### CODE REVIEW

**Comment by otherRick** [comment]
Task linked: [CU-2y2ptjv [FE] Dashboard Integration](https://app.clickup.com/t/2y2ptjv)

**Review by Djonatanabreu — APPROVED** [review]

**Comment by danielanthonyl** [comment]
Task linked: [CU-301p2fw [FE]  [Feat] add axios auth instance](https://app.clickup.com/t/301p2fw)

**Review by danielanthonyl — COMMENTED** [review]
I cannot approve since I created the pull request itself. But here is a ✅

**Review by otherRick — APPROVED** [review]

File: src/components/BottomSheet/CategoryListBottomSheet/_components/CategoryListItem/index.tsx [hunk 1]
```diff
@@ -4,17 +4,25 @@ import { ICategoryListItemProps } from './types';
...
+    dispatch(bottomSheetControl({ visible: false }));
+  };
   return (
     <Swipeable
       containerStyle={styles.swipeableContainer}
       overshootLeft={false}
       overshootRight={false}
       renderRightActions={props => <SwipeRight {...props} {...{ id }} />}>
-      <RectButton style={styles.itemTextContainer}>
+      <RectButton onPress={onPress} style={styles.itemTextContainer}>
```
Comment (danielanthonyl) [inline src/components/BottomSheet/CategoryListBottomSheet/_components/CategoryListItem/index.tsx:25]: aslkdfhjsdlkfgjdslkfjsdlkjfoslkdj

Comment (danielanthonyl) [reply src/components/BottomSheet/CategoryListBottomSheet/_components/CategoryListItem/index.tsx:25]: resolvi no commit tal... 6f5a7c2a849b49dd7032ff56bd12c36812294d6d

**Pull request merged**